front/.eslintrc.js

```diff
@@ -0,0 +1,157 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    jest: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react-hooks/recommended',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'app/tsconfig.json',
+    sourceType: 'module',
+  },
+  plugins: [
+    'eslint-plugin-react',
+    'jsx-a11y',
+    '@typescript-eslint',
+    '@typescript-eslint/tslint',
+  ],
+  ignorePatterns: ['app/containers/App/constants.js'],
+  rules: {
+    '@typescript-eslint/member-delimiter-style': [
+      'off',
```

Comment:
Not sure I understand this config. Does it mean we don't apply the rule, or only on single line type defs ? We used to have semi-columns on all type definitions, last line included, shall we keep the habit (I don't have a strong opinion there, but maybe good to keep a standard)

Reply:
Nice catch, Sara! I am guessing this turned off for some reason in the transformation of the config. I used a [tool](https://github.com/typescript-eslint/tslint-to-eslint-config) for this but it's not perfect. 
Anyhow, fixed now 🙂